ImageFolder/cards/card.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>components</title>
+    <!-- <link rel="stylesheet" href="/card1.css"> -->
+    <link rel="stylesheet" href="/components/card.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.10.0/css/all.min.css" integrity="sha512-PgQMlq+nqFLV4ylk1gwUOgm6CtIIXkKwaIHp/PAIWHzig/lKZSEGKEysh0TCVbHJXCLN7WetD8TFecIky75ZfQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+<body>
+
+    <div class="megaContainer">
+
```

Comment:
class names should not be camelCase 
use `mega-container`

Reply:
removed that mega container and fixed spacing . Can you approve please?